### DISCUSSION TITLE
Playback Toolbar streamlining (Play/Pause)

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6491] [Ideas]
The Playback Toolbar features both play and pause buttons, I think it would make sense to combine these into a single button. As you cannot play while playing or pause while paused, it does not make sense for both of these buttons to be visible at all times. Take video players as an example, they have a single unified button that switches based on whether or not the video is playing.

Additionally, I believe that it may make more sense to change the behavior of the Loop button to be a toggle rather than its current behavior. If an animation is playing and the Loop button is pressed, the animation will pause. This is a little counter-intuitive as the aspect of looping is more of a condition than an action in of itself.

Once the play and pause buttons are combined and the Loop button is moved, you end up with a cleaner and more logical layout
![image](https://user-images.githubusercontent.com/42905003/82764666-5f0f8100-9e08-11ea-81b7-a4218143762a.png)

**Comment by DarrenTAnims** [comment]
Personally, I'd like to see the pause button hidden at the start, and then play and loop play buttons disappear and the pause button show when you play with either. But I've also added a request elsewhere for a pingpong play button, so maybe if this change were to go ahead, perhaps that could be included?

Before pressing play:
![image](https://user-images.githubusercontent.com/28575907/82810850-61fd8680-9e87-11ea-837b-33087653e425.png)

After pressing any of the 3 play buttons:
![image](https://user-images.githubusercontent.com/28575907/82810856-6629a400-9e87-11ea-940f-d73ba7b5fd3f.png)

**Comment by RodneyBaker** [comment]
@DarrenTAnims There can be a pretty solid argument against disappearing icons but at least in your example the location of the play and pause icon doesn't change position.  The disappearing icons does make the last two icons (next frame and last frame) change locations so that would be a little painful for those that often use those options.

I was having a similar discussion on discord and current am trying to use the shortcut key 'P' to play and pause.  If I could get that committed to muscle memory life would be good.

@WolfinABowl I do like the idea of having the play and pause buttons be interchangeable at the same location.

**Comment by RodneyBaker** [comment]
In #3306 this request was discussed with some additional interest.
Ultimately, a recommendation was to hide (or be able to hide) the Pause button as the Play button currently functions as both Play and Pause.
